wazimap_ng/utils.py

```diff
@@ -586,3 +588,27 @@ def pivot(d, order):
     nested = nest(rearranged)
 
     return nested
+
+
+def detect_encoding(buffer):
+    detector = UniversalDetector()
+    for line in buffer:
+        detector.feed(line)
+        if detector.done: break
+    detector.close()
+    return detector.result["encoding"]
+
+
+def get_encoding_and_wrapper_file(buffer):
```

Comment:
this function is doing too much.
can we refactor this to just return the wrapper file.
we can use the detect_encding method before we call this wrapper function and pass the encoding.

so it would be `get_stream_reader(buffer, encoding='utf-8')`
and just return the `wrapper_file` 
we can create a new method for the column replacement
`clean_columns(wrapper_file)` or so. 

repeating function calls is not that bad but we can dry up the actual implementation.

Reply:
Okay, @milafrerichs . It sounds sensible. I'll change it on Monday.

Reply:
I changed it @milafrerichs. Please check it out: https://github.com/OpenUpSA/wazimap-ng/pull/169/commits/b78d5db7750dbf34045394fa0477deabb7dd2c20